fastlane/Fastfile

```diff
@@ -237,6 +237,24 @@ reflected in `Info.plist` during building."
 
   end
 
+  desc "Runs `Swiftformat` and `Swiftlint`. If running on CI, Swiftformat will fail if there are changes\
+  to be done."
+  lane :run_linter do |options|
+
+    desc "Running Swiftformat"
+
+    changes = `git diff --shortstat`
+    `cd .. && swiftformat . && cd fastlane`
```

Comment:
You can run just: `swiftformat ..` just in case we decide to change the folder named `fastlane`